docs/src/main/sphinx/release/release-419.md

```diff
@@ -0,0 +1,64 @@
+# Release 419 (5 June 2023)
```

Comment:
We've used `Jun` instead of `June` in the past release notes.

Reply:
@ryandoowan can you check and maybe send a small PR to fix this .. cc @colebow